File: java/src/main/java/com/a65apps/kalimullinilnazrafilovich/interactors/details/ContactDetailsRepository.java
package com.a65apps.kalimullinilnazrafilovich.interactors.details;

import com.a65apps.kalimullinilnazrafilovich.entities.ContactDetailsInfo;

import io.reactivex.rxjava3.core.Single;

public interface ContactDetailsRepository {
    Single<ContactDetailsInfo> getDetailsContact(String id);
}

File: java/src/main/java/com/a65apps/kalimullinilnazrafilovich/interactors/details/ContactDetailsRepository.kt
package com.a65apps.kalimullinilnazrafilovich.interactors.details

import com.a65apps.kalimullinilnazrafilovich.entities.ContactDetailsInfo
import kotlinx.coroutines.flow.Flow

interface ContactDetailsRepository {
    fun getDetailsContact(id: String): Flow<ContactDetailsInfo>
}
